files/package-plugin.sh

```diff
@@ -7,21 +7,22 @@ PROJECT_NAME="$1"
 mkdir -p "${PROJECT_NAME}"
 
 for path in `git ls-files | grep -v .gitignore`; do
-	file=$(basename "$path")
-	dir=$(dirname "$path")
+        file=$(basename "$path")
+        dir=$(dirname "$path")
 
-	mkdir -p "${PROJECT_NAME}/${dir}"
-	cp "${path}" "${PROJECT_NAME}/${path}"
+        mkdir -p "${PROJECT_NAME}/${dir}"
+        cp "${path}" "${PROJECT_NAME}/${path}"
 done
 
 pushd "${PROJECT_NAME}"
 
 composer install -qno --no-dev --ignore-platform-reqs
 
-if [ ! -f .sw-zip-blacklist ]; then
-    cp -r /tmp/.sw-zip-blacklist.dist .sw-zip-blacklist
+if [ ! -f .sw-zip-allowlist ]; then
+    cp -r /Users/a.wink/scripts/.zip-allowlist .zip-allowlist
 fi
-sed -i -e "s#^#${PROJECT_NAME}/#" .sw-zip-blacklist
+
+comm -2 -3 <(ls -a) <(sort .zip-allowlist) | tail +2 | xargs rm -rf
+rm -rf .zip-allowlist
 
 popd
-zip -rq $PROJECT_NAME.zip $PROJECT_NAME --exclude @$PROJECT_NAME/.sw-zip-blacklist $PROJECT_NAME/.sw-zip-blacklist
```

Comment:
Why remove the ZIP creation altogether? I like the possibility to download the finished plugin ZIP from the artifacts directly. Maybe both the folder and the ZIP can be kept as artifacts?